src/gc/stats.d

```diff
@@ -0,0 +1,54 @@
+/**
+ * Utility to simplify calculating `core.memory.GC.Stats`
+ *
+ * Copyright: Copyright Digital Mars 2018.
+ * License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ */
+module gc.stats;
+
+/**
+    Embeds `GC.Stats` instance and adds few simple wrapper methods
+    to maniplate it while doing some sanity checks.
+ */
+package(gc) struct GCStatsTracker
+{
+    @nogc nothrow:
+
+    import core.memory;
+    GC.Stats stats;
+
+    /// Record new memory added to the GC pool from OS
```

Comment:
The style for Ddoc functions is:
```
/*********
 * Record new memory added to the GC pool from the OS.
 * Params:
 *   bytes = amount of new memory added
 */
```
Yes, it looks rather redundant, but tools expect this format.